src/GeneralStateTestsFiller/VMTests/vmTests/push0GasFiller.yml

```diff
@@ -0,0 +1,56 @@
+# EIP-3855: PUSH0 instruction - test for gas price
+push0Gas:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentBaseFee: '10'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 10000000
+      code: ''
+      nonce: '0'
+      storage: {}
+
+    0000000000000000000000000000000000000100:
+      balance: 0
+      code: ':raw 0x5f'
+      nonce: '0'
```

Comment:
Test for gasPrice 
(sstore 0 (gas))
Push0
(Sstore 0 (sub (gas) (sload 0)))

Make it irrelevant from other gas that is affected by other evm changes

Reply:
Changed to record gas usage in storage.

Reply:
Why not use memory for it?

Reply:
this is a version using MSTORE/MLOAD: https://gist.github.com/hugo-dc/4740c46803771c701b12d417a348f3b8#file-push0gasfiller-yml-L21

Looks very similar to the SSTORE/SLOAD version, although the expect section looks cleaner.

Reply:
memory is also ok as long as the result is checed in sstore. and memory does not intersect with some other region that is in use